src/ResourceManagement/EventHub/Generated/EventHubManagementClient.cs

```diff
@@ -25,6 +25,11 @@ namespace Microsoft.Azure.Management.EventHub.Fluent
     /// </summary>
     public partial class EventHubManagementClient : FluentServiceClientBase<EventHubManagementClient>, IEventHubManagementClient, IAzureClient
     {
+        /// <summary>
+        /// The base URI of the service.
+        /// </summary>
+        //public System.Uri BaseUri { get; set; }
```

Comment:
remove this if its not needed

Reply:
These are auto generated code. I followed up with this https://github.com/Azure/azure-libraries-for-net/pull/639.